allauth/socialaccount/providers/ebay/provider.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class EBayAccount(ProviderAccount):
+    pass
+
+
+class EBayProvider(OAuth2Provider):
+    id = "ebay"
+    name = "EBay"
+    account_class = EBayAccount
+
+    def extract_uid(self, data):
+        # NOTE: Ensure that the user_id exists
```

Comment:
This comment is a bit unclear. It refers to `user_id` while the code uses `userid`. Also, what is meant by this comment? Who is to ensure this?

Reply:
I honestly was not to sure if I was meant to extract user_id or userid. That's why that comment is there. I was following Dwolla and Dropbox implementation as an example.

Reply:
Looking at their docs over here https://developer.ebay.com/api-docs/commerce/identity/overview.html they only seem to be offering `userId` ?

Reply:
That looks right, will be updated next push